lms/static/sass/shared/_fields.scss

```diff
@@ -0,0 +1,69 @@
+// lms - shared - fields
+// ====================
+
+
+.u-field {
+    padding: $baseline 0;
+    border-bottom: 1px solid $gray-l5;
+
+    .message-error {
+        color: $alert-color;
+    }
+
+    .message-validation-error {
+        color: $warning-color;
+    }
+
+    .message-in-progress {
+        color: $gray-d2;
+    }
+
+    .message-success {
+        color: $success-color;
+    }
+}
+
+.u-field-readonly {
+    input[type="text"],
+    input[type="text"]:focus {
+        background-color: transparent;
+        padding: 0px;
+        border: none;
+        box-shadow: none;
+    }
+}
+
+.u-field-title {
+    width: flex-grid(3, 12);
+    display: inline-block;
+    color: $dark-gray1;
+    vertical-align: top;
+    margin-bottom: 0;
```

Comment:
@symbolist, please do not do this.  The default behavior (clickable labels) is a benefit for everyone and should not be overridden, especially for field inputs.  @marcotuts can you tell me where its happening where you don't think it should be happening?  If it's happening on read-only fields like username, it's cool to remove pointer-events from that use case, but definitely not all of them.